### PR TITLE
fix: display hex-decoded currency in AMM transaction table

### PR DESCRIPTION
**Pull request by mvadari**
## High Level Overview of Change

Update the AMM transaction table to use the Currency component to consolidate currency formatting logic

### Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Refactor (non-breaking change that only restructures code)
- [x] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [x] Translation Updates

### CODE REVIEW

**Review by ckniffen — COMMENTED** [review]
Can you add a test.

**Review by pdp2121 — APPROVED** [review]

File: src/containers/shared/components/TxToken.tsx [hunk 1]
```diff
@@ -18,12 +21,28 @@ function getTokenPair(
...
+      ) : undefined
 
     if (first && second) {
-      return first + (type === 'Payment' ? ' for ' : ' and ') + second
+      return (
+        <Trans
+          i18nKey={
+            type === 'Payment'
+              ? 'token_for_transaction_swap'
+              : 'token_for_transaction'
```
Comment (JST5000) [inline src/containers/shared/components/TxToken.tsx:38]: I'm not really sure what `token_for_transaction` means - is there a more descriptive label that could be used?

**Review by mvadari — COMMENTED** [review]
I can't approve because this was originally my PR but ✅

**Review by JST5000 — APPROVED** [review]
LGTM

**Pull request merged**